src/content/blog/2025-01-31-ilp-packet-lifecycle.mdx

```diff
@@ -0,0 +1,96 @@
+
+Finally, it creates an _outgoing payment_ against the sending wallet address by acquiring consent from its owner to charge them with particular terms such as its amount and its currency.
+The prior incoming payment (or the quote, if created), is provided during this process and becomes associated with the outgoing payment.
+Once that is complete, these Rafiki instances can begin to fulfill the terms of this payment using Interledger.
+
+On a more technical point, ILP packets may be involved before an outgoing payment is even created. During the acqusition of a _quote_ from the sending Rafiki instance, it will send a ILP packet accross the network to the destination. This packet will need to traverse each hop on the network as each node's fees and rates won't be known to the sender without doing so.
+
+_For a more on Open Payments, consider reading the [Simple Open Payments Guide](https://interledger.org/developers/blog/simple-open-payments-guide/) on this blog._
+
+### Preparing the Packet
+Now that the sending Rafiki instance has an outgoing payment created against one of its wallet addresses, it can pass the terms of that payment into an Interledger payment. First, it uses the incoming payment associated with it (or the quote if it was created) to acquire an ILP address for the receiver and the shared secret, which is used to generate a [_condition_](/developers/rfcs/stream-protocol/#6-condition-and-fulfillment-generation).
+A condition is a hash that can only be generated with a secret that is available only to the receiver and the sender.
+Using the receiving ILP address, the condition, the receiver, and send amount described in the outgoing payment, Rafiki creates a connection over ILP to the receiver using [STREAM](/developers/rfcs/stream-protocol/).
+STREAM is a transport-layer protocol for Interledger that is responsible for breaking a payment down into the aforementioned packets and sends them _en masse_ until the payment is fulfilled.
+
+For each packet, STREAM handles them using a two-phase execution process that is similar to the request-response process that HTTP packets sent over the internet use.
+
+![Two-Phase Transfer Diagram](/developers/img/blog/2025-01-31/2p-transfer.png)
+
+In the first phase, the sender creates packets known as [Prepare packets](/developers/rfcs/interledger-protocol/), which each contain some fraction of the total send amount configured by each peer along the ILP network path, the condition generated using SPSP, the final destination of the packet in the network, and an expiration time for the packet.
+All of these packets are sent across the network to the receiver, and must be fulfilled or rejected by the receiver in the second phase in order to be considered successfully sent.
+
+![ILP Prepare Packet Specification](/developers/img/blog/2025-01-31/ilp-prepare-spec.png)
```

Comment:
I wonder if this image could use a title to indicate what the image is of. It doesn't explicitly explain that this is the structure of a prepare packet. And the same for reject and fulfill.

Other than that LGTM.